src/SSHDebugPS/UI/ViewModels/ContainerPickerViewModel.cs

```diff
@@ -162,6 +162,11 @@ private void RefreshContainersListInternal()
                     containers = DockerHelper.GetRemoteDockerContainers(connection, Hostname, out totalContainers);
                 }
 
+                foreach(DockerContainerInstance container in containers)
+                {
+                    container.Platform = DockerHelper.GetContainerPlatform(Hostname, container.Name);
```

Comment:
Are we throwing all the containers away if we cant get a platform? What happened to going back to `docker info -f "{{.Server.OS}}"`?

Reply:
also, we shouldn't throw them away. we should support an "Unknown" option in that field

Reply:
maybe think about an enum here that has Unknown, Windows and Linux. This way if you have to do comparisons later with it, you aren't doing string comparisons.